eth/api.go

```diff
@@ -605,6 +605,9 @@ func (api *DebugAPI) SetTrieFlushInterval(interval string) error {
 	if err != nil {
 		return err
 	}
+	if t <= 0 {
+		return errors.New("invalid value, TrieFlushInterval should be positive")
+	}
```

Comment:
Why though? Right now, if you set a negative value, it will just accept it. So you can set `-1`, and later on, it will always flush. It will just be like "archive mode". 

As far as I can tell, the code will not do anything wrong, such as crash, on negative values.

Reply:
Also, this code prevents `=0`, which means that this change makes it not possible to set it to archive-mode (always flush). Sure, in practice you can set it to `1ns`, and it will be archive mode, but still... Why?

Reply:
Sorry, I didn’t think of this use case earlier. So maybe we can add a comment to explain the usage of 0 and negative numbers.

Reply:
revert && add comment